secure_message/services/party_service.py

```diff
@@ -10,18 +10,30 @@
 
 
 class PartyService:
+
+    users_cache = dict()
```

Comment:
Should these be prefixed with underscores (i.e.: `_users_cache`) if they're specific only to this class?